pages/common/if.md

```diff
@@ -1,27 +1,28 @@
 # if
 
 > Simple shell conditional.
+> See also: `test`, `[`.
 
-- Echo a different thing depending on a command's success:
+- Executed two different commands based on a condition:
 
-`if {{command}}; then echo "success"; else echo "failure"; fi`
+`if {{condition}}; then {{echo "true"}}; else {{echo "false"}}; fi`
 
-- Full if syntax:
+- Check if a variable is defined:
 
-`if {{condition}}; then echo "true"; else echo "false"; fi`
+`if [[ ! -z {{$VARIABLE}} ]]; then {{echo "defined"}}; else {{echo "not defined"}}; fi`
 
-- List available if conditions:
+- Check if a file exists:
 
-`help test`
+`if [[ -f {{path/to/file}} ]]; then {{echo "true"}}; else {{echo "false"}}; fi`
 
-- Test if a given variable is empty:
+- Check if a directory exists:
 
-`if [[ -z $GIT_BRANCH ]]; then echo "true"; else echo "false"; fi`
+`if [[ -d {{path/to/directory}} ]]; then {{echo "true"}}; else {{echo "false"}}; fi`
 
-- Test if a file exists:
+Check if a file or directory ([e]ntry) exists:
 
-`if [[ -e {{filename}} ]]; then echo "true"; else echo "false"; fi`
+`if [[ -e {{path/to/file_or_directory}} ]]; then {{echo "true"}}; else {{echo "false"}}; fi`
```

Comment:
I think the`-e` here refers to "exists", more than to "entry"...

Reply:
Hmm...you must be right 👍🏻